internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -220,20 +220,24 @@ func (e *EnvStackConfig) StackName() string {
 	return NameForEnv(e.in.App.Name, e.in.Name)
 }
 
+type transformParameterFunc func(new *cloudformation.Parameter, old *cloudformation.Parameter) *cloudformation.Parameter
```

Comment:
same on all function signatures
```suggestion
type transformParameterFunc func(new, old *cloudformation.Parameter) *cloudformation.Parameter
```